src/components/paragraph/sul-featured-collection.tsx

```diff
@@ -36,49 +46,63 @@ const SulFeaturedCollection = ({headerId, headline, link, cards, styles, fullWid
   return (
     <section className="centered relative" ref={ref} {...props}>
       {headline && (
-        <header className="mb-40 flex flex-row items-center justify-between gap-16">
+        <header
+          className={clsx("mb-40", {
+            "flex flex-row items-center justify-between gap-16": styles?.display_heading_gradient,
+          })}
+        >
           <h2 id={headerId} className="mb-0 shrink-0">
             {headline}
           </h2>
+          {styles?.display_heading_gradient && <HeaderGradientLine />}
         </header>
       )}
 
       <div className="relative @container">
-        <ul className="list-unstyled grid gap-xl @7xl:grid-cols-3">
+        <ul className="list-unstyled grid gap-x-90 gap-y-30 sm:grid-cols-2 lg:grid-cols-3">
           {cards.map(card => (
             <li key={card.id}>
-              <CollectionCard
-                header={card.suCardHeader}
-                superHeader={card.suCardSuperHeader}
-                body={card.suCardBody?.processed}
-                link={card.suCardLink}
-                image={card.suCardMedia?.__typename === "MediaImage" ? card.suCardMedia : undefined}
-                videoUrl={card.suCardMedia?.__typename === "MediaVideo" ? card.suCardMedia.mediaOembedVideo : undefined}
-              />
+              {card.__typename === "ParagraphStanfordCard" ? (
+                <CollectionCard
+                  header={card.suCardHeader}
+                  superHeader={card.suCardSuperHeader}
+                  body={card.suCardBody?.processed}
+                  link={card.suCardLink}
+                  image={card.suCardMedia?.__typename === "MediaImage" ? card.suCardMedia : undefined}
+                  videoUrl={
+                    card.suCardMedia?.__typename === "MediaVideo" ? card.suCardMedia.mediaOembedVideo : undefined
+                  }
+                />
+              ) : card.__typename === "ParagraphStanfordStatCard" ? (
+                <StatCardParagraph paragraph={card} />
+              ) : null}
```

Comment:
Instead of doing these type of ternary operators, break it out for easier readability.

```
{card.__typename === "foo" && <Foo />}
{card.__typename === "bar" && <Bar />}
```